### PR TITLE
leafletjs.com

### PR DESCRIPTION
**Pull request by theel0ja**
https://github.com/Leaflet/Leaflet/issues/6147

### CODE REVIEW

**Comment by mourner** [comment]
@theel0ja do we need this rule if I enforce HTTPS in the GitHub Pages settings?

**Comment by theel0ja** [comment]
@mourner No.

**Comment by J0WI** [comment]
The rule is needed anyway: https://www.eff.org/https-everywhere/faq#why-does-https-everywhere-include-rules-for-sites-like-paypal-that-already-require-https-on-all-their-pages

File: src/chrome/content/rules/leafletjs.com.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="leafletjs.com">
+        <target host="leafletjs.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/leafletjs.com.xml]: Please also add `www.leafletjs.com ` and a custom redirect to  `^`.

**Pull request closed without merging**